ReactAndroid/build.gradle

```diff
@@ -213,6 +213,8 @@ android {
     compileSdkVersion 23
     buildToolsVersion "23.0.1"
 
+    useLibrary 'org.apache.http.legacy'
```

Comment:
Why is this necessary now but wasn't before? We shouldn't need to use any new libraries for this module.